docs/syllabus.md

```diff
@@ -0,0 +1,70 @@
+# QMK Syllabus
+
+This page helps you build up your QMK knowledge by introducing the basics first and guiding you to understanding all the concepts you need to know to be proficient with QMK.
+
+# Beginning Topics
+
+If you read nothing else you should read the documents in this section. After reading the [Tutorial](newbs.md) you should be able to create a basic keymap, complie it, and flash it to your keyboard. The remaining documents will flesh out your knowledge of these basics.
+
+* **Learn How To Use QMK Tools**
+    * [Tutorial](newbs.md)
+    * [CLI](cli.md)
+    * [GIT](newbs_git_best_practices.md)
+* **Learn About Keymaps**
+    * [Layers](feature_layers.md)
+    * [Keycodes](keycodes.md)
+        * The full list of keycodes you can use. Note that some may require knowledge found in the Intermediate or Advanced Topics.
+* **Configuring IDEs** - Optional
+    * [Eclipse](other_eclipse.md)
+    * [VSCode](other_vscode.md)
+
+# Intermediate Topics
+
+These topics start dig into some of the features that QMK supports. You don't have to read all of these documents, but some of the documents in the Advanced Topics section won't make sense if you skip over some of these.
+
+* **Learn How To Configure Features**
+    * Configuration Overview <!-- FIXME(skullydazed/anyone): write this document -->
```

Comment:
I think we should either remove this until the document is not written or note it somehow, so the user will not find an "unclickable link".